### ISSUE TITLE
Load/import configuration without the use of the extension

### ISSUE THREAD

**Issue by fpietka** [tony/tmuxp#160]
Would be nice to be able to load (or import) configuration without having to write the extension:
```
$ tmuxp load myproject
```
instead of
```
$ tmuxp load myproject.yaml
```
also for import:
```
$ tmuxp import tmuxinator myproject
```

**Comment by tony** [comment]
It'd be better if tmuxp could just load from tmuxinator configs.  the good news its import gets us a lot of the way there.

**Comment by fpietka** [comment]
worked like a charm for me 👍

**Comment by tony** [comment]
#164

It is more flexible than ever.  In addition to being able to load up configs by name from any dir. You can also target directories that contain ``.tmuxp.{json,yml,yaml}`` relatively and absolutely.

If you want to test it, its in the https://github.com/tony/tmuxp/tree/click branch.

**Comment by tony** [comment]
I don't know of teamocil/tmuxinator's conversions are good enough to load directly.  I don't think the behavior would be 100%, it's probably at about 80-85% atm. I need to double check things like ``pre:`` get translated and behave like they would on tmuxp.

**Comment by tony** [comment]
@fpietka landed in v1.1.0 on pypi now

**Comment by oschrenk** [comment]
Sweet. I was actually about my own fish script to do this. Thanks for the great work!
